ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-SPINE2.yml

```diff
@@ -158,8 +158,6 @@ management_interfaces:
     ip_address: 192.168.200.102/24
     gateway: 192.168.200.5
     type: oob
```

Comment:
When you remove the change to default behavior, please set the `ignore_tcam_system_profiles` on this switch to test the behavior.

Reply:
Sure